CRM/Contact/BAO/Group.php

```diff
@@ -370,7 +370,15 @@ public static function create(&$params) {
           self::setIsActive($childValue, (int) ($params['is_active'] ?? 1));
         }
       }
+
+      // get current parents for removal if not in the list anymore
+      $currentParentGroupIDs = [];
+      $parents = CRM_Core_DAO::getFieldValue('CRM_Contact_DAO_Group', $params['id'], 'parents');
+      if (!empty($parents)) {
+        $currentParentGroupIDs = explode(',', $parents);
+      }
```

Comment:
Slight simplification of code:
```suggestion
      $parents = CRM_Core_DAO::getFieldValue('CRM_Contact_DAO_Group', $params['id'], 'parents');
      $currentParentGroupIDs = $parents ? explode(',', $parents) : [];
```